### PR TITLE
Fix #11063: Exchange hack for call to update_scroll

### LINKED ISSUE #11063: Scenery window scrolling issue
**OS:** Windows 10 (x64)
**Version:** 0.2.5
**Commit/Build:** da2ba0b

When you scroll down a long scenery tab, then switch to a different tab, the scroll position often persists. This can lead to a scenery tab appearing empty if the scroll position is below the last item in the tab.

This doesn't seem to happen all the time, but often enough to be an annoyance.

- [ ] Reproducible in RCT2 (vanilla)?
- [ ] Specific to multiplayer?

**Steps to reproduce:**
1. Open scenery window
2. Open a scenery tab with a lot of items and scroll down
3. Switch to a different scenery tab, observe that it will scroll down to where you were in the previous tab

### PR DESCRIPTION
**Pull request by jeysbach**
Fixes #11063 

Removed an old hack and everything seems to be working correctly now.

### CODE REVIEW

**Comment by Gymnasiast** [comment]
It would be nice if one-liner was the solution. I do want to test it though, and also check if it doesn't regress the issue that the hack aimed to solve.

An aside: it's not a problem since we will just squash it, but it looks like you have set up your local Git with no e-mail address, or with one that is different from the one on Github. (You should be able to have it work properly by setting it to `jeysbach@users.noreply.github.com` if you don't want to give away your actual e-mail address.

**Comment by jeysbach** [comment]
Right, thanks for the heads up. E-mail thing should be fixed now.

I was also surprised this worked, but it seems like the hack stopped working at some point and update_scroll was not called anymore.

**Comment by Gymnasiast** [comment]
OK, this seems to work for me. I have also checked if it regresses #3210, which it doesn't.

Now, two things left:

- I'd like @Umdlye to test it.
- Could you amend the changelog? `Fix: [#11063] Scrolling position persists when switching tabs in the scenery window.`

**Comment by Umdlye** [comment]
Works correctly here too :)

File: distribution/changelog.txt [hunk 1]
```diff
@@ -9,6 +9,7 @@
 - Fix: [#11005] Company value overflows.
 - Fix: [#11027] Third color on walls becomes black when saving.
 - Improved: [#11157] Slimmer virtual floor lines.
+- Fix: [#11063] Scrolling position persists when switching tabs in the scenery window.
```
Comment (Gymnasiast) [inline distribution/changelog.txt]: The changelog is grouped by change type (fix/feature/improved/etc.). This needs to be moved up one row.

**Pull request merged**